File: 6-kyu/replace-with-alphabet-position/replace-with-alphabet-position.js
function alphabetPosition(text) {
  let splitText = text.replaceAll(' ', '').toLowerCase().split('')
  let alpha = 'abcdefghijklmnopqrstuvwxyz';
 let mapped = splittext.map(char => {
   let indexed = alpha.indexOf(char)
   return indexed != -1 ? indexed + 1 : '';
 })
 }
​

File: 6-kyu/replace-with-alphabet-position/replace-with-alphabet-position.java
class Kata {
    static String alphabetPosition(String text) {
      // Implement me! :)
      //String[] textArr = text.split("");
      StringBuilder result = new StringBuilder(); 
      for(char i : text.toLowerCase().toCharArray()){
     //  int index = text.indexOf(i);
        if(i >= 'a' && i<= 'z'){
          int position = i - 'a' + 1;
          result.append(position).append(" ");
        }
      }
      return result.toString().trim();
    }
}
​